include/seqan3/alignment/pairwise/align_pairwise.hpp

```diff
@@ -181,7 +182,7 @@ constexpr auto align_pairwise(sequence_t && sequences,
                                      | views::chunk(traits_t::alignments_per_vector);
 
     // Create a two-way executor for the alignment.
-    detail::alignment_executor_two_way executor{indexed_sequence_chunk_view,
+    detail::blocking_algorithm_executor executor{indexed_sequence_chunk_view,
```

Comment:
I liked the prefix-based name before. What about `algorithm_executor_blocking`?

Side question: Can the executor only work with algorithms? (or is everything that takes an executor an algorithm?) Just wondering if the `algorithm` in the name is necessary.

Reply:
Well the semantics are clear, since it calls something with an input. That is in a very generic case an algorithm. We could call it function executor but does it make it better? Also the context right now is for algorithms so why should it be named differently? 
I rename it to algorithm_executor_blocking though, to be more conistent with other namings.